Build from src instructions

I tried to follow the instructions to build from src.

1. Download go
2. Setup go, verified GOPATH was correct, all good there
3. Clone repo into $GOPATH/gravitational/teleport
```
cd $GOPATH
mkdir gravitational
git clone `enter url here`
```
4. cd teleport
5. make
6. Errors upon errors about missing dependencies. (below)

I rode the struggle bus for a while. Once I tried `go get github.com/gravitational/teleport` and built it from the resulting `$GOPATH/src/github.com/gravitational/teleport` directory, everything was fine. This was much easier than trying to use git to clone the the repo and build that way. It also helps verify that you have go setup correctly.


Errors from make
```
$ make
make -f version.mk setver
make[1]: Entering directory `/home/synovia/work/gravitational/teleport'
make[1]: Leaving directory `/home/synovia/work/gravitational/teleport'
go build -o build/teleport -i -ldflags -w ./tool/teleport
tool/teleport/main.go:34:2: cannot find package "github.com/Sirupsen/logrus" in any of:
        /usr/local/go/src/github.com/Sirupsen/logrus (from $GOROOT)
        /home/synovia/work/src/github.com/Sirupsen/logrus (from $GOPATH)
tool/teleport/main.go:28:2: cannot find package "github.com/gravitational/teleport/lib/config" in any of:
        /usr/local/go/src/github.com/gravitational/teleport/lib/config (from $GOROOT)
        /home/synovia/work/src/github.com/gravitational/teleport/lib/config (from $GOPATH)
tool/teleport/main.go:29:2: cannot find package "github.com/gravitational/teleport/lib/defaults" in any of:
        /usr/local/go/src/github.com/gravitational/teleport/lib/defaults (from $GOROOT)
        /home/synovia/work/src/github.com/gravitational/teleport/lib/defaults (from $GOPATH)
tool/teleport/main.go:30:2: cannot find package "github.com/gravitational/teleport/lib/service" in any of:
        /usr/local/go/src/github.com/gravitational/teleport/lib/service (from $GOROOT)
        /home/synovia/work/src/github.com/gravitational/teleport/lib/service (from $GOPATH)
tool/teleport/main.go:31:2: cannot find package "github.com/gravitational/teleport/lib/sshutils/scp" in any of:
        /usr/local/go/src/github.com/gravitational/teleport/lib/sshutils/scp (from $GOROOT)
        /home/synovia/work/src/github.com/gravitational/teleport/lib/sshutils/scp (from $GOPATH)
tool/teleport/main.go:32:2: cannot find package "github.com/gravitational/teleport/lib/utils" in any of:
        /usr/local/go/src/github.com/gravitational/teleport/lib/utils (from $GOROOT)
        /home/synovia/work/src/github.com/gravitational/teleport/lib/utils (from $GOPATH)
tool/teleport/main.go:35:2: cannot find package "github.com/gravitational/trace" in any of:
        /usr/local/go/src/github.com/gravitational/trace (from $GOROOT)
        /home/synovia/work/src/github.com/gravitational/trace (from $GOPATH)
make: *** [teleport] Error 1

```

--- a/README.md
+++ b/README.md
@@ -36,7 +36,13 @@ teleport start
 
 You need to have Golang `v1.7` or newer. 
 
-Clone this repository into `$GOPATH/gravitational/teleport` and run `make`. 
+Run 
+
+1. `go get github.com/gravitational/teleport`
+2. `cd $GOPATH/src/github.com/gravitational/teleport`
+3. `make`
+
+If the build was successful the binaries are here: `$GOPATH/src/github.com/gravitational/teleport/build`
 
 You'll have to create `/var/lib/teleport` directory and then you can start 
 Teleport as a single-node cluster in development mode: `build/teleport start -d`

